brats/transformations.py

```diff
@@ -73,3 +76,87 @@ def hist_match(source: np.ndarray, template: np.ndarray) -> np.ndarray:
                                     template_values)
         result[positive] = interp_t_values[bin_idx]
         return result
+
+
+class NiftiOrderTransformation:
+    """
+    Changes dimensions order from nifti (H,W,D,C) to torch convention (C,D,W,H).
+    It can be applied to both ``torch.Tensor`` or ``numpy.ndarray``.
+    """
+
+    def __call__(self, img):
```

Comment:
Since when we are not using type hints. I generally think that those `isinstance` can be replaced with typehints,

Reply:
I forgot to put them there. I agree with the replacement